test/integration/pages/home.py

```diff
@@ -50,11 +48,18 @@ def lockie(self):
         """Lockie image locator."""
         return self.find_element(*self._lockie_locator).text
 
-    def add_entry(self):
-        """Add an entry into the lockbox."""
+    def create_new_entry(self, site_name='', url='', username='',
+                         password='', note=''):
+        """Create and save a new entry."""
         current_entries = len(self.entries)
         self.find_element(*self._new_entry_locator).click()
-        self.find_element(*self._save_entry_locator).click()
+        entry = self.entries[0].click()
+        entry.set_site_name(site_name)
+        entry.set_website(url)
+        entry.set_username(username)
+        entry.set_password(password)
+        entry.set_note(note)
```

Comment:
Is there a reason we would only set 1 of these?

Reply:
Do mean can a user have multiple notes per entry? At this time, no.

Reply:
No, I am asking if we would only use ```set_username``` for example, or would it be all of them every time?

Reply:
I follow now, thanks. I think for this first iteration I was thinking yes. The next round of work I want to move things around a bit more and could see this method returning an Entry object and the individual test touches the setter methods. Do you see this as blocking or can this happen in a separate pr?

Reply:
Not it's not blocking. I think we could iterate on this though. Maybe we don't need so many ```set_``` methods.

Reply:
Agreed. I felt unclean doing so.